base/rational.jl

```diff
@@ -375,8 +375,8 @@ function ^(x::Rational, n::Integer)
 end
 
 ^(x::Number, y::Rational) = x^(y.num/y.den)
-^{T<:AbstractFloat}(x::T, y::Rational) = x^(convert(T, y.num / y.den))
-^{T<:AbstractFloat}(x::Complex{T}, y::Rational) = x^(convert(T, y.num / y.den))
+^{T<:AbstractFloat}(x::T, y::Rational) = x^(convert(T,y.num)/convert(T,y.den))
+^{T<:AbstractFloat}(x::Complex{T}, y::Rational) = x^(convert(T,y.num)/convert(T,y.den))
```

Comment:
No, because then it will call `^(x::Number, y::Rational)` instead.  (Remember, Julia always calls the most specific method.)